src/path_rewriting.rs

```diff
@@ -235,6 +236,13 @@ pub fn rewrite_paths(
     }
     let to_ignore_globset = glob_builder.build().unwrap();
 
+    glob_builder = GlobSetBuilder::new();
```

Comment:
it's the same code to build globs for ignore so you should add a function to do that stuff.